src/librbd/CMakeLists.txt

```diff
@@ -227,12 +227,14 @@ if(WITH_RBD_RWL OR WITH_RBD_SSD_CACHE)
   if(WITH_RBD_RWL)
     set(librbd_internal_srcs
       ${librbd_internal_srcs}
+      cache/pwl/RWLLogEntry.cc
```

Comment:
Nit: should be lexicographically after `Replicated...`

Reply:
Done